src/main/java/net/pms/dlna/LibMediaInfoParser.java

```diff
@@ -175,8 +175,8 @@ public synchronized static void parse(DLNAMediaInfo media, InputFile inputFile,
 						}
 
 						value = MI.Get(video, i, "Format_Profile");
-						if (!value.isEmpty() && media.getCodecV() != null && media.getCodecV().equals(FormatConfiguration.H264)) {
-							media.setAvcLevel(getAvcLevel(value));
+						if (media.getCodecV() != null && isNotBlank(value)) {
```

Comment:
Calling that "avcLevel" when all the video profiles (mpeg-4 visual, 3gp...) was logged without distinction, was of no mean to me...